### DISCUSSION TITLE
How many papers do you need in a dataset?

### DISCUSSION THREAD

**Discussion by SagevdBrand** [asreview/asreview#570] [Q&A] [answered]
Does the size of your dataset for the performance of ASReview matter? Is there maybe a minimum number of papers that I would need to upload to benefit time-wise from ASReview?

**Comment by boer0107-zz** [comment, marked as answer]
Size matters for ASReview, but we can't put a number to this.
The bigger the dataset, the more time ASReview will save you. But it also depends on the number of prior inclusions that you have; when you have a small dataset with little prior knowledge ASReview will save you less time. It may also depend on the inclusion-exclusion ratio and even on the contents on your specific dataset and how well ASReview performs.
